### ISSUE TITLE
[Configuration] Can't add ARM using UI (#1523)

### ISSUE THREAD

**Issue by nguerrera** [dotnet/sdk#851]
(Moved from dotnet/roslyn-project-system#1523)

* File -> New Project -> Console App (.NET Core)
* Click on platform dropdown in toolbar -> Configuration manager
  * Notice that ARM is not a choice
* Type in ARM manually and leave "create project platforms" checked -> OK
  * Get popup error that projects do not support ARM
* Uncheck "create project platforms" -> OK
* Edit project file, add ARM configurations manually:
``` xml
  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM'" />
  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'" />
```
* Go back to configuration manager, attempt to set ARM project configuration to match ARM solution configuration.
  * No errors, but it just resets it back to Sln=ARM -> Project=Any CPU

Workaround:

* Edit sln manually in addition to editing sln manually.

@davkean @RaulPerez1 Is there a capability that drives whether a project can target ARM from VS' perspective?

**Comment by nguerrera** [comment]
@davkean
> Yeah MSBuild property drives this, AvailablePlatforms I think it's called.

**Comment by nguerrera** [comment]
@nguerrera
> That works. I don't want to advertise support for something that hasn't been used in RTM. And you can add $(AvailablePlatforms);ARM to your own targets or csproj as workaround. I'll move this to SDK for post 1.0.

**Comment by nguerrera** [comment]
We should consider adding ARM to AvailablePlatforms based on the target framework.

**Comment by marcpopMSFT** [comment]
Was fixed in 16.10
